layouts/partials/blocks/awards.html

```diff
@@ -0,0 +1,269 @@
+{{ with site.Data.awards }}
+  <div class="h-auto w-full bg-beige pb-16">
+    <div class="mx-auto flex max-w-4xl items-center px-4 pb-4 pt-16">
+      <svg class="mr-4 h-8 w-9" aria-hidden="true" style="fill: #b6b6b6;">
+        {{ partial "helper/svg.html" "trophy" }}
+      </svg>
+      <h2 class="text-4xl font-bold">Awards</h2>
+    </div>
+
+    <div class="mx-auto max-w-4xl px-4">
+      <div style="border-top: 2.5px solid black; width: 100%;"></div>
+
+      <div class="flex w-full">
+        <div
+          id="years-padding"
+          class="font-bold"
+          style="padding: 15px 20px 15px 25px; border-right: 2.5px solid black;"
+        >
+          <ul class="year-list">
+            {{ $years := slice }}
+            {{ range . }}
+              {{ $years = $years | append .year }}
+            {{ end }}
+            {{ $sortedYears := uniq $years | sort }}
+            {{ $count := len $sortedYears }}
+
+            {{ range $index := seq 0 (sub $count 1) }}
+              <li>{{ index $sortedYears (sub (sub $count 1) $index) }}</li>
+            {{ end }}
+          </ul>
+        </div>
+
+        {{ $awards := site.Data.awards }}
+        <script>
+          document.addEventListener("DOMContentLoaded", () => {
```

Comment:
Doesn't need to be rewritten now, but there's probably a simpler way to write this with Alpine.js. Probably just put an `x-data="{ activeYear: '2024' }"` on the container and then `x-show="$el.dataset.year === activeYear" data-year="{{ .year }}"` on the main panel and `@click="activeEl = $el.dataset.year" data-year="{{ .year }}"` on the sidebar links.